style.css

```diff
@@ -123,3 +123,188 @@ p {
   line-height: 32px;
   text-align: center;
 }
+
+.card {
+  height: 350px;
+  background-color: #ebf0ee;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-around;
+  align-content: center;
+  border-radius: 6px;
+  display: grid;
+  margin-top: 20px;
+  margin-right: 25px;
+  margin-left: 25px;
+}
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038812147) from the previous reviewer. 

![Screenshot from 2022-12-04 01-23-37](https://user-images.githubusercontent.com/60083437/205458250-02b68546-ea54-4e8d-a30d-d3eed05a38b0.png)